### PR TITLE
Readme.md

### PR DESCRIPTION
**Pull request by janitastic**
## Subject:
Readme Branch Review
## Reason for Pull Request:
Submit readme file contributions
## Changes Implemented:
- Contributors
- About Section
- Technologies Used
- Instructions
- Future Addtions
- Deployment Page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,39 @@
 # Intention-Timer
+
+### Contributors
+
+Jani Palominu [janitastic](https://github.com/janitastic)
```
Comment (janitastic) [inline README.md]: Correct spelling - Jani Palomin**o**
Put Jani Palomino in the []

File: README.md [hunk 2]
```diff
@@ -1 +1,39 @@
...
+
+Lauren Kessell [LKessell](https://github.com/LKessell)
+
+Andrey Bermudez [Andrey-1992](https://github.com/Andrey-1992)
+
+
+# About
+The Intention Timer app seeks to provide a chronometer to help the user to organize, achieve and keep tracking of their goals. With this app, each user will have the ability to set customize time frames and add specifications to work on, in three different categories:
```
Comment (janitastic) [inline README.md]: great use of chronometer!

Comment (janitastic) [reply README.md]: Update the second sentence to - 

With this app the user has the ability to set a study, meditation, or exercise goal and add a custom description of what they'd like to accomplish during the timeframe they set.

File: README.md [hunk 3]
```diff
@@ -1 +1,39 @@
...
+ **Study**
```
Comment (janitastic) [inline README.md]: remove, already mentioned above
Duplicate of #13

File: README.md [hunk 4]
```diff
@@ -1 +1,39 @@
...
+ **Meditation**
```
Comment (janitastic) [inline README.md]: remove, already mentioned above
Duplicate of #13

File: README.md [hunk 5]
```diff
@@ -1 +1,39 @@
...
+ **Exercises**
```
Comment (janitastic) [inline README.md]: remove, already mentioned above
Duplicate of #13

File: README.md [hunk 6]
```diff
@@ -1 +1,39 @@
...
+
+
+ ## Technologies Used
+
+   Javascript
```
Comment (janitastic) [inline README.md]: JavaScript with a capital S - more specifically Vanilla JavaScript

File: README.md [hunk 7]
```diff
@@ -1 +1,39 @@
...
+
+   HTML
+
+   CSS
+
+# Instructions
+1. When the user first loads the app, they have to select one of the three categories to work on it.
```
Comment (janitastic) [inline README.md:18]: Update to - 
When the user first loads the app, they have to select one of the three categories they'd like to set a timer for: study, meditate, or exercise.

File: README.md [hunk 8]
```diff
@@ -1 +1,39 @@
...
+2. The user have to write a description about their activity during the timer.
```
Comment (janitastic) [inline README.md]: Update to - 
After a category is selected, the user has to write a description about what they'd like to accomplish during the timeframe they set.

File: README.md [hunk 9]
```diff
@@ -1 +1,39 @@
...
+3. The user must specify in the section of minutes and second the time that he would like to work, after that he is ready so start the activity.
+4. In the next window he can verify is the information is correct, and get ready to set the timer on.
+5. When the users complete the activity, they would have the possibility to log it and keep track of their process.
```
Comment (janitastic) [inline README.md]: replace possibility with **option**

File: README.md [hunk 10]
```diff
@@ -1 +1,39 @@
...
+6. Finally they can start a new activity clicking the **CREATE A NEW ACTIVITY** button.
```
Comment (janitastic) [inline README.md]: add 
Finally they can start a new activity **and timer by**

File: README.md [hunk 11]
```diff
@@ -1 +1,39 @@
 # Intention-Timer
+
+### Contributors
+
+Jani Palominu [janitastic](https://github.com/janitastic)
+
+Lauren Kessell [LKessell](https://github.com/LKessell)
```
Comment (janitastic) [inline README.md]: Put Lauren Kessell []

File: README.md [hunk 12]
```diff
@@ -1 +1,39 @@
...
+
+Andrey Bermudez [Andrey-1992](https://github.com/Andrey-1992)
```
Comment (janitastic) [inline README.md]: Put Andrey Bermudez in []

File: README.md [hunk 13]
```diff
@@ -1 +1,39 @@
...
+3. The user must specify in the section of minutes and second the time that he would like to work, after that he is ready so start the activity.
+4. In the next window he can verify is the information is correct, and get ready to set the timer on.
+5. When the users complete the activity, they would have the possibility to log it and keep track of their process.
+6. Finally they can start a new activity clicking the **CREATE A NEW ACTIVITY** button.
+
+   ### Deployed page:
+     [Intention Timer](https://andrey-1992.github.io/Intention-Timer/)
+
+ ## Future Additions
+Future additions to the Intention Timer will allow the users to be able to re-do specific cards, with the possibility to set different time frames and modify specifications. Besides the user will have the possibility to selected the most important and  relevant cards and save them as *Favorites*.
```
Comment (janitastic) [inline README.md]: Update second sentence to -
The user will also have the ability to select the most important and relevant cards and save them as *Favorites*.

**Review by janitastic — COMMENTED** [review]
I made a few edit/comments. Can you make these changes and then request a merge with main? This will add in the video gif.

**Comment by LKessell** [comment]
Looks wonderful!

**Pull request merged**